consensus/src/network_interface.rs

```diff
@@ -76,9 +79,19 @@ pub type ConsensusNetworkEvents = NetworkEvents<ConsensusMsg>;
 #[derive(Clone)]
 pub struct ConsensusNetworkSender {
     network_sender: NetworkSender<ConsensusMsg>,
+    connections: Option<Arc<RwLock<HashMap<PeerId, SupportedProtocols>>>>,
 }
 
+/// Supported RPC protocol.
+pub const RPC: &[ProtocolId] = &[ProtocolId::ConsensusRpc];
+/// Supported Direct Send protocol.
+pub const DIRECT_SEND: &[ProtocolId] = &[
+    ProtocolId::ConsensusDirectSendCbor,
+    ProtocolId::ConsensusDirectSend,
+];
```

Comment:
I'd comment that they're in priority (highest first) order, as it applies later if they're in the wrong order.